db/structure.sql

```diff
@@ -132,6 +133,8 @@ CREATE TYPE public.review_state AS ENUM (
 
 SET default_tablespace = '';
 
+SET default_table_access_method = heap;
```

Comment:
What is this change for?

Reply:
I didn't manually make this. I don't know what it is or how it got set 😕.

Reply:
@nicofh what PostgreSQL version are you on?

Reply:
13.0

Reply:
Cool, thanks. I don't think this should pose any problem, but in any case we can disregard `structure.sql` diffs not related to our changesets.

Reply:
I've experienced some problems with this line before, because it doesn't work with versions of postgres earlier than 12. We may still not be interested in avoiding this problem, but I think that if we don't know what this line is doing, we may want to keep it away from the file.